sdk/newrelicobservability/Azure.ResourceManager.NewRelicObservability/api/Azure.ResourceManager.NewRelicObservability.netstandard2.0.cs

```diff
@@ -55,566 +55,566 @@ public partial class NewRelicMonitorResourceCollection : Azure.ResourceManager.A
+        public static implicit operator Azure.ResourceManager.NewRelicObservability.Models.NewRelicObservabilityUsageType (string value) { throw null; }
+        public static bool operator !=(Azure.ResourceManager.NewRelicObservability.Models.NewRelicObservabilityUsageType left, Azure.ResourceManager.NewRelicObservability.Models.NewRelicObservabilityUsageType right) { throw null; }
         public override string ToString() { throw null; }
     }
-    public partial class SwitchBillingContent
+    public partial class NewRelicObservabilityUserInfo
     {
-        public SwitchBillingContent(string userEmail) { }
-        public string AzureResourceId { get { throw null; } set { } }
+        public NewRelicObservabilityUserInfo() { }
+        public string Country { get { throw null; } set { } }
+        public string EmailAddress { get { throw null; } set { } }
+        public string FirstName { get { throw null; } set { } }
+        public string LastName { get { throw null; } set { } }
+        public string PhoneNumber { get { throw null; } set { } }
+    }
+    public partial class NewRelicObservabilityVmExtensionPayload
+    {
+        internal NewRelicObservabilityVmExtensionPayload() { }
+        public string IngestionKey { get { throw null; } }
+    }
+    public partial class NewRelicObservabilityVmInfo
+    {
+        internal NewRelicObservabilityVmInfo() { }
+        public string AgentStatus { get { throw null; } }
+        public string AgentVersion { get { throw null; } }
+        public Azure.Core.ResourceIdentifier VmId { get { throw null; } }
+    }
+    public partial class NewRelicOrganizationResourceData : Azure.ResourceManager.Models.ResourceData
+    {
+        public NewRelicOrganizationResourceData() { }
+        public Azure.ResourceManager.NewRelicObservability.Models.NewRelicObservabilityBillingSource? BillingSource { get { throw null; } set { } }
         public string OrganizationId { get { throw null; } set { } }
-        public Azure.ResourceManager.NewRelicObservability.Models.PlanData PlanData { get { throw null; } set { } }
-        public string UserEmail { get { throw null; } }
+        public string OrganizationName { get { throw null; } set { } }
+    }
+    public partial class NewRelicPlan
+    {
+        public NewRelicPlan() { }
+        public Azure.ResourceManager.NewRelicObservability.Models.NewRelicObservabilityBillingCycle? BillingCycle { get { throw null; } set { } }
+        public System.DateTimeOffset? EffectiveOn { get { throw null; } set { } }
+        public string PlanDetails { get { throw null; } set { } }
+        public Azure.ResourceManager.NewRelicObservability.Models.NewRelicObservabilityUsageType? UsageType { get { throw null; } set { } }
+    }
+    public partial class NewRelicPlanResourceData : Azure.ResourceManager.Models.ResourceData
```

Comment:
Avoid to use `Resource` as suffix, so here `NewRelicPlanData` should be better

Reply:
Updated